### ISSUE TITLE
Add Gym Info

### ISSUE THREAD

**Issue by rezz3x** [AHAAAAAAA/PokemonGo-Map#532]
It would be nice if you could add Gym Information.

Like:
- Pokemon's that are in it
- Level
- CP of the Pokemon's

**Comment by alexpantyukhin** [comment]
I will try to implement this.

**Comment by Lunat1q** [comment]
You need to send additional request server to get Gym info.

**Comment by alexpantyukhin** [comment]
Yeah) I know. And it seems that handler for this request already exists https://github.com/AHAAAAAAA/PokemonGo-Map/blob/develop/pogom/app.py#L19

**Comment by Lunat1q** [comment]
But, that can add additional load on nearly dead niantic servers, should
this be done?

ср, 20 июля 2016, 16:05 Alexander Pantyukhin <notifications@github.com>:

**Comment by rezz3x** [comment]
You could add a argument, if you really need this Feature like:
-gi 
For Gym Info.

**Comment by alexpantyukhin** [comment]
I don't think that it would be load the server. Actually Gyms object are already parsed and puted into the db. And it make an additional request wouldn't be so hard. Agree with @rezz3x but not sure that it's really need.

**Comment by alexpantyukhin** [comment]
Do we have some icon for pokemon gym? I'm not sure maybe there is some suitable icon for that?

**Comment by FlorianChedemail** [comment]
@Lunat1q Those servers are already dying and the main problem is this poorly optimized "step" algorithm which does redundant requests. The covering is far from optimal.
